CMakeLists.txt

```diff
@@ -191,12 +197,20 @@ include_directories("${PROJECT_SOURCE_DIR}/3rdParty/tinyobjloader/")
 
 find_package(LibElf REQUIRED)
 include_directories(${LIBELF_INCLUDE_DIRS})
+include(FindSDL)
```

Comment:
Can we set these to only required for the main build step? That would fix the issue with convenience steps like CPPcheck now breaking because we don't install the full suite of packages on the build runner for that.

Reply:
see below if that's acceptable